examples/gcoap/credentials.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2018 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       tlsman test application (PSK and ECC keys)
```

Comment:
The `tlsman` module of @rfuentess was never merged. We should change the wording here to not confuse newcomers.

Reply:
Edited the @brief and description.